src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2247,7 +2248,9 @@ def aks_update_credentials(cmd, client, resource_group_name, name,
                        name, parameters)
 
 
-def aks_check_acr(cmd, client, resource_group_name, name, acr):
+def aks_check_acr(cmd, client, resource_group_name, name, acr, node_name=None):
+    if not acr.endswith(CONST_ACR_DOMAIN_NAME):
+        acr = acr + CONST_ACR_DOMAIN_NAME
```

Comment:
Append acr domain suffix to the name, aligns to the usage of similar option `--attach-acr` in other commands like `aks create` and `aks update`.